chemistry/rosetta/pose.py

```diff
@@ -0,0 +1,79 @@
+"""
+This package contains classes responsible for loading and dumping PyRosetta
+Pose objects.
+"""
+
+from __future__ import print_function
+
+from chemistry.exceptions import RosettaError, RosettaWarning
+from chemistry.periodic_table import AtomicNum, Mass
+from chemistry.structure import Structure
+from chemistry.topologyobjects import Atom, Bond
+
+try:
+    from rosetta import Pose, AtomID
+except ImportError:
+    RosettaWarning('Cannot find the PyRosetta module.')
```

Comment:
Usually what I do is `pass` (or set what I imported to `None`), and then raise an `ImportError` when I try to use functionality in a module that is not available.  See what I do [here](https://github.com/ParmEd/ParmEd/blob/master/chemistry/openmm/reporters.py) with OpenMM.  The only reason I used a decorator was because I had many functions that depended on OpenMM and only wanted to put the check in one place.  In this case, Rosetta is only used once so the decorator approach is probably overkill.